README.md

```diff
@@ -62,3 +62,30 @@ To keep commit messages consistent, we use [gitmoji-cli](https://www.npmjs.com/p
 
   2. Run `npm run build-uikit` which performs the following:
   3. Run `npm run publish-uikit` to publish latest version to npm
+
+# Testing
+
+Tests use React Testing Library and Jest
+Jest uses ts-jest not babel so emotion needs pragma at top
+ie. /\*_ @jsxImportSource @emotion/react _/
+
+https://kentcdodds.com/blog/common-mistakes-with-react-testing-library
+
+`screen.debug` is incredibly useful to view jsdom output
+
+# Emotion key points
+
+css from @emotion/react does not return the computed class name string. The function returns an object containing the computed name and flattened styles. The returned object is understood by emotion at a low level and can be composed with other emotion based styles inside of the css prop, other css calls, or the styled API.
+
+https://emotion.sh/docs/css-prop
+
+Styled api takes precedence over css
+
+# Recommended usage
+
+Use `css` for consistency
+hard to define what takes precedence when we mix styled and css
+
+Themes will merge with same versions of emotion
+UIKit theme is namespaced
+Please use as last theme provider
```

Comment:
"last" may require a bit more elaboration... does this mean the provider needs to be nested as a child of all other (external) theme providers?
e.g. Arranger theme provider. Curious about the results for that overlap, if it was to ever happen 🤔

Reply:
that makes sense. followup question: if your provider inherits `{ colors: blue, uikit: null }` from a parent provider?
and if it gets overwritten, how do the components respond? e.g. `theme.uikit.colors.blue`?